spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.feature 'Users Index View', type: :feature do
+  scenario 'User views users index page' do
+    user1 = User.create(name: 'Alice', photo: 'https://example.com/alice.jpg')
+    user1.posts.create(title: 'Post 1', text: 'First post by Alice')
+    user1.posts.create(title: 'Post 2', text: 'Second post by Alice')
+    user2 = User.create(name: 'Bob', photo: 'https://example.com/bob.jpg')
+    user2.posts.create(title: 'Post 3', text: 'First post by Bob')
+    visit users_path
+    expect(page).to have_selector('.container')
+    expect(page).to have_selector('.user_info', count: 4)
```

Comment:
- [ ] As mentioned by the previous reviewer the below test is failing, Please make sure that all tests pass without any errors.

![image](https://github.com/Obimbo07/tdn/assets/83803414/678ec360-f90b-4730-81b6-c2dd7c0089ff)

Reply:
Hello,

When I submit my tests they pass but I don't know why on reviewers side it doesn't. The previous reviewer told me to submit this video in the comment section.

https://github.com/Obimbo07/tdn/assets/118368849/bb2d1a69-9fac-4adc-aa29-f4a2c6ab8e0e